File: administrativoService/src/main/java/br/com/blas/hemoblas/service/EstabelecimentoSaudeService.java
package br.com.blas.hemoblas.service;

import java.util.List;

import br.com.blas.hemoblas.model.EstabelecimentoSaude;

public interface EstabelecimentoSaudeService {

	public List<EstabelecimentoSaude> listarEstabelecimentosSaudePorMunicipioAndTipoUnidade(Integer municipioId,
			Integer tipoUnidadeId);

}

File: hemoblas/src/main/resources/static/js/service/estabelecimentoSaudeService.js
angular.module("hemoblas").service(
		"estabelecimentoSaudeService",
		function($http, config) {

			var tipoUnidade = {
				CENTRO_ATENCAO_HEMOTERAPIA : 69
			};

			this.getHemocentrosPorMunicipio = function(municipioId) {

				return $http.get(config.baseUrlAdministrativoSecure + "/estabelecimentosSaude/busca/municipioId=" + municipioId
						+ "&tipoUnidadeId=" + tipoUnidade.CENTRO_ATENCAO_HEMOTERAPIA);

			};

		});